tests/FSharp.Test.Utilities/CompilerAssert.fs

```diff
@@ -289,10 +289,7 @@ module rec CompilerAssertHelpers =
         member x.ExecuteTestCase assemblyPath (deps: string[]) =
             AppDomain.CurrentDomain.add_AssemblyResolve(ResolveEventHandler(fun _ args ->
                 deps
-                |> Array.tryFind (fun (x: string) -> Path.GetFileNameWithoutExtension x = args.Name)
-                |> Option.orElseWith (fun () -> 
-                    deps 
-                    |> Array.tryFind (fun (x: string) -> args.Name.StartsWith(Path.GetFileNameWithoutExtension(x) + ", Version")))
+                |> Array.tryFind (fun (x: string) -> Path.GetFileNameWithoutExtension x = AssemblyName(args.Name).Name)
```

Comment:
```suggestion
                |> Array.tryFind (Path.GetFileNameWithoutExtension >> (=) (AssemblyName(args.Name).Name))
```

Reply:
@Happypig375 , I am afraid I don't find the suggestion easier to read.  I am quite old :-)